personality_insights/ApplicationBuilder/personality-insights-twitter-widget.erb

```diff
@@ -0,0 +1,108 @@
+
+<script>
+  
+  function displayModel(model) {
+    var $model_container = $(".user_model").empty(); 
+    
+    // model_container.append(JSON.stringify(model));
+    
+    if (model["word_count_message"] !== undefined) {
+      $("<div>").append(model["word_count_message"]).append("<br>").appendTo($model_container);  
+    }
+         
+      
+    for (var i in model["tree"]["children"]) {
+      var major_category = model["tree"]["children"][i];
+      $("<center>").append($("<h2>").append(major_category["name"])).appendTo($model_container);
+            
+      var $table = $("<table>").addClass("um");
+      $table.append("<tr><th>Value</th><th>Watson Score</th></tr>");
+      
+      for (var j in major_category["children"]) {
+        var sub_category = major_category["children"][j];
+                              
+        for (var k in sub_category["children"]) {
+          var sub_sub_category = sub_category["children"][k];
+        
+          var $row = $("<tr>");
+            
+          if (sub_sub_category["percentage"] > 0.8) {
+            $row.addClass("interesting");
+          }
+          
+          var $name = $("<td>").append(sub_sub_category["name"]);
+          var $score = $("<td>").append((sub_sub_category["percentage"] * 100).toFixed(3));
+          
+          $row.append($name).append($score);
+          $table.append($row);
+        }
+      }    
+
+      $model_container.append($table).append("<br>");
+    }
+  }
+  
+  
+  $(function() {
+    $(".go").click(function() {
+      $(".watson-status").show();
+      
+      var endpoint = "/AppBuilder/endpoint/personality_insights_twitter";
```

Comment:
We should have a comment in the code here, as well as text in the readme, that suggests a new value for 'endpoint' if using the proxy.